### DISCUSSION TITLE
Export/Import to move from Play Store to F-Droid

### DISCUSSION THREAD

**Discussion by tomcat-xx** [tughi/aggregator-android#64] [General]
Is there a way to take all settings from Play Store to F-Droid version? Feeds are in OPML file but what's about tags with all the rules? Have rootet phone, can access all but did not find it so far.

**Comment by tughi** [comment]
If you have a rooted phone it's easier to move the database file to the new app: 

    cp /data/data/com.tughi.aggregator.next/databases/data.* /data/data/com.tughi.aggregator/databases/

**Reply by nickyg666** [reply]
Is there any way to backup the database (specifically read entries and older entries) without root?

**Reply by tughi** [reply]
It was possible in the past to backup and restore the data using the adb command.

Here is a nice guide: https://gist.github.com/AnatomicJC/e773dd55ae60ab0b2d6dd2351eb977c1
